app/scripts/controllers/preferences.js

```diff
@@ -31,7 +31,7 @@ export default class PreferencesController {
   constructor(opts = {}) {
     const initState = {
       frequentRpcListDetail: [],
-      useBlockie: false,
+      useBlockie: true,
```

Comment:
@dragana8 So we have made a decision and there is a specification change for this ticket. Can you remove this change so that it defaults to false. The unit tests will also need modification, but otherwise I think the rest of the PR is good.

Reply:
@Gudahtt @danjm in the ticket https://github.com/MetaMask/metamask-extension/issues/13264 was specified that **All new users have Blockies as default** 
I just reverted that change.

Reply:
@danjm @Gudahtt 

Please see the Notion (Acceptance criteria at the bottom): https://www.notion.so/Make-default-account-icons-the-same-on-mobile-and-extension-jazzicon-or-blockie-4188ebb3412f4c849c89405331a695b8

Reply:
I'm updating the Acceptance criteria to "All new users have Jazzicon as default".